packages/revocation/src/constants/constants.ts

```diff
@@ -3,4 +3,4 @@ export const RevocationRegOffChainAddr = '0x3A859706c2491993Cf07760bf68F40B10946
 export const RevocationRegOnChainAddr = '0xb155d0ff939de83a5f26676fEcEd7f208C39055C';
 export const ENSRegistryAddr = '0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac';
 export const ClaimManagerAddr = '0xC3dD7ED75779b33F5Cfb709E0aB02b71fbFA3210';
-export const ERC1056Addr = '0x84d0c7284A869213CB047595d34d6044d9a7E14A';
```

Comment:
@nichonien where is this address coming from?

Reply:
In the initial days I got confused between `IdentityManager` and `DIDRegistry`, and so I had used IdentityManager's address in place of DIDRegistry's address.